### PR TITLE
add dockerfiles to support openshift CI

### PR DESCRIPTION
**Pull request by hhpatel14**

### CODE REVIEW

File: build/Dockerfile.bundle
Comment (dymurray) [inline build/Dockerfile.bundle:6]: This needs to be `oadp-operator`

File: build/custom-ci-Dockerfile
Comment (shawn-hurley) [inline build/custom-ci-Dockerfile:5]: Are we using molecule tests for oadp-operator?

Comment (hhpatel14) [reply build/custom-ci-Dockerfile:5]: Yes, based on what I discussed with Dylan. But I am always open to suggestions.

Comment (dymurray) [reply build/custom-ci-Dockerfile:5]: @shawn-hurley I asked Hiteshwari to integrate openshift CI with calling out to molecule so that we could leverage it like mig-operator if desired.

It is likely that we change this to simply be an e2e test suite written in golang... so I will hold off on merging until we've made that decision.

Comment (dymurray) [inline build/custom-ci-Dockerfile:5]: remove these, add `yum -y install make`

Comment (dymurray) [reply build/custom-ci-Dockerfile:5]:
```
go get -u github.com/onsi/ginkgo/ginkgo
```

Comment (shawn-hurley) [reply build/custom-ci-Dockerfile:5]: There may already be a testing base image that we could use for ginkgo tests

Comment (dymurray) [reply build/custom-ci-Dockerfile:5]: Good idea Shawn

File: build/custom-ci-Dockerfile
Comment (dymurray) [inline build/custom-ci-Dockerfile:4]: According to https://onsi.github.io/ginkgo/ we may also need:
```
go get github.com/onsi/gomega/...
```

I would add it to be safe

**Review by dymurray — COMMENTED** [review]
LGTM once we add the last `go get` command

**Review by dymurray — APPROVED** [review]

**Pull request merged**
